PhysObjectExtractor/python/poet_cfg.py

```diff
@@ -78,9 +79,10 @@
 
 process.mypvertex = cms.EDAnalyzer('VertexAnalyzer')
 process.mytracks= cms.EDAnalyzer('TrackAnalyzer')
+process.mygenparticle= cms.EDAnalyzer('GenParticleAnalyzer',input_particle = cms.vstring("electron","muon","photon","tau"))
 
 process.TFileService = cms.Service(
     "TFileService", fileName=cms.string("myoutput.root"))
 
-process.p = cms.Path(process.myevents+process.myelectrons+process.mymuons+process.myphotons+process.myjets+process.mymets+process.mytaus+process.mytrigEvent+process.mypvertex+process.mytracks)
+process.p = cms.Path(process.myevents+process.myelectrons+process.mymuons+process.myphotons+process.myjets+process.mymets+process.mytaus+process.mytrigEvent+process.mypvertex+process.mytracks+process.mygenparticle)
```

Comment:
You know, this mygenparticle module only should run if the `isData` flag is **not** True, otherwise it will crash.  I do not know the best way to do this, but maybe the simplest will be to define the Path also based on the isData flag (like it is done for the the jet module).  It is kind of dirty, but I can't find a way to declare an empty EDAnalyzer module to make it cleaner.